go/Blind 75/arrayAndHashing/217-Contains-Duplicate.go

```diff
@@ -0,0 +1,13 @@
+package arrayAndHashing
```

Comment:
nit: do you mind matching the naming convention for all these files & folders to be the same as the cpp dir? https://github.com/neetcode-gh/leetcode/tree/main/cpp/neetcode_150

the dir and file names should all be `snake_case`